GsaGHTests/3_Components/2_Geometry/EditElement1dTests.cs

```diff
@@ -289,5 +289,31 @@ public void InvalidPropertyElementTypeCombination3() {
       comp.Params.Output[0].CollectData();
       Assert.Empty(comp.RuntimeMessages(Grasshopper.Kernel.GH_RuntimeMessageLevel.Error));
     }
+
+    [Fact]
+    public void ElementReleasesAreCorrectWhenBoolValuesAssignedAsInput() {
+      GH_OasysComponent comp = ComponentMother();
+      var startReleaseInput = new GsaBool6(true, true, true, false, false, false);
+      var endReleaseInput = new GsaBool6(false, false, false, true, true, true);
+      ComponentTestHelper.SetInput(comp, new GsaBool6Goo(startReleaseInput), 7);
+      ComponentTestHelper.SetInput(comp, new GsaBool6Goo(endReleaseInput), 8);
+      GsaBool6 startReleaseOutput = ((GsaBool6Goo)ComponentTestHelper.GetOutput(comp, 7)).Value;
+      GsaBool6 endReleaseOutput = ((GsaBool6Goo)ComponentTestHelper.GetOutput(comp, 8)).Value;
+      Assert.Equal(startReleaseInput, startReleaseOutput);
+      Assert.Equal(endReleaseInput, endReleaseOutput);
+    }
```

Comment:
can you test its input in a separate test, no need to compare them at the same time